software/react-native/recommended-libraries.md

```diff
@@ -1,20 +1,20 @@
 # Recommended Basic Stack (Current Standard)
 
```

Comment:
any discussions about `ignite` ? I think how to stand up a project here would be great, maybe you guys talk about this in another section so if so, can ignore this comment